Zuly-Montaño/tarea2/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Revista</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+    <main class="container">
+        <header>
+            <section class="titulo-flex">
+                <article>
+                    <h1>Petronio Alvarez</h1>
+                </article>
+            </section>
+        </header>
+
+        <section class="description">
+            <article>
+                <p>23 años de festival de la musica y cultura de nuestro hermoso pacifico</p>
+                <p>Cali, Valle del Cauca</p>
+            </article>
+        </section>
+
+        <section class="text">
+            <aside class="text1">
+                <article>
+                    <p>variedad de platos tipicos afrocolombianos</p>
+                </article>
+            </aside>
+            <aside>
+                <article class="containerImagen">
```

Comment:
@zuly7 , apply the class to the parent tag , in this case the aside tag (which should be a section) and remove the article tag.